lib/hydra/derivatives/processors/full_text_extraction.rb

```diff
@@ -0,0 +1,59 @@
+module Hydra::Derivatives::Processors
+  # Extract the full text from the content using Solr's extract handler
+  class FullTextExtraction < Processor
+
+    def process
+      output_file_service.call(extract, directives)
+    end
+
+    ##
```

Comment:
Do we want all of the methods below this point (*sans* `connection_url`) to be public? Or could they be protected?